process/block/preprocess/transactionsV2.go

```diff
@@ -185,7 +185,7 @@ func (txs *transactions) processTransaction(
 		}
 
 		mbInfo.processingInfo.numBadTxs++
-		log.Trace("bad tx", "error", err.Error(), "hash", txHash)
+		log.Debug("bad tx", "error", err.Error(), "hash", txHash)
```

Comment:
Intended.

Reply:
no. It will fill up the txLogs with too many things. Revert.

Reply:
Reverted.